src/format/table.rs

```diff
@@ -88,9 +97,9 @@ impl TableView {
                 entries[row].truncate(max_num_of_columns);
             }
 
-            headers.push("...".to_string());
+            headers.push("…".to_string());
```

Comment:
To keep things maximally portable, we've been trying to stick to ASCII (and away from Unicode). Keeping this as the expanded form should be okay.